File: examples/python/job/job_details.py
import http.client

conn = http.client.HTTPSConnection("exenzo.nl")

headers = {
    'x-authorization': "ORGANISATION_API_TOKEN",
    'content-type': "application/json",
    'cache-control': "no-cache",
    }

conn.request("GET", "/api/v1/job/{job_id}", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: examples/ruby/job/job_details.rb
require 'uri'
require 'net/http'

url = URI("https://exenzo.nl/api/v1/job/{job_id}")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true
http.verify_mode = OpenSSL::SSL::VERIFY_NONE

request = Net::HTTP::Get.new(url)
request["x-authorization"] = 'ORGANISATION_API_TOKEN'
request["content-type"] = 'application/json'
request["cache-control"] = 'no-cache'

response = http.request(request)
puts response.read_body